client/service/src/client/wasm_override.rs

```diff
@@ -104,22 +104,19 @@ impl From<WasmOverrideError> for sp_blockchain::Error {
 /// Scrapes WASM from a folder and returns WASM from that folder
 /// if the runtime spec version matches.
 #[derive(Clone, Debug)]
-pub struct WasmOverride<E> {
+pub struct WasmOverride {
 	// Map of runtime spec version -> Wasm Blob
 	overrides: HashMap<u32, WasmBlob>,
-	executor: E,
```

Comment:
I don't know much about this part of the code, but having the generic here could ensure that the same type is used every time any of its method is called. Is it required here ?

Reply:
Not really, we just need the executor at startup to get the runtime versions.